### ISSUE TITLE
errors on "ttoggle -s"

### ISSUE THREAD

**Issue by gissf1** [pigmonkey/nmtrust#14]
Hello, I recently discovered your project and installed it via AUR on my laptop.  I added my networks and services to the configuration files as documented.  It seems to be working as expected, but when I run `ttoggle -s` at the command line, I got errors.

```bash
$ ttoggle -s
Systemd system units:
● ntpd.service - Network Time Service
     Active: active (running) since Thu 2026-09-10 16:17:19 CDT; 2min 10s ago
● sshd.service - OpenSSH Daemon
     Active: active (running) since Thu 2026-09-10 16:17:15 CDT; 2min 14s ago
Systemd user units:
id: ‘’: no such user
usage: sudo -h | -K | -k | -V
usage: sudo -v [-ABkNnS] [-g group] [-h host] [-p prompt] [-u user]
usage: sudo -l [-ABkNnS] [-g group] [-h host] [-p prompt] [-U user]
            [-u user] [command [arg ...]]
usage: sudo [-ABbEHkNnPS] [-C num] [-D directory]
            [-g group] [-h host] [-p prompt] [-R directory] [-T timeout]
            [-u user] [VAR=value] [-i | -s] [command [arg ...]]
usage: sudo -e [-ABkNnS] [-C num] [-D directory]
            [-g group] [-h host] [-p prompt] [-R directory] [-T timeout]
            [-u user] file ...
```

In my `trusted_units` file, I do not have any "user" units defined; only "system" units are defined.

The final failure happens in function `user_toggle()` when `extract_user()` returns an empty string and therefore `unit_users` is an empty string, it is not caught here and still follows the "else" section on line 59, triggering the above errors.

The call stack at the point of failure is:
- extract_user()
- user_toggle()
- status()

I noticed a few issues:
- on line 49 and 50, `$line` should probably be `$2` since you're passing it in as the second argument in all calls and not using the argument otherwise.  Also, referencing a variable (`$line`) that should probably be considered local to the calling function scope is asking for future issues.
- the `status()` function does not check if `$TRUSTED_USER_UNITS` is empty before iterating it with a while loop.  This results in a single loop iteration where `$line` is an empty string, causing the issue above.

Explaining in another way, `status()` is missing the `if [ -n ... ]` check used in `start()` and `stop()`.

Adding the `if` also provides the opportunity to display a message like "No systemd user units defined in trusted_services configuration file" when that is the case.

I can possibly provide a PR to patch this, but hopefully the above detailed explanation is enough for you to resolve the issue as you see fit.

Thanks for providing this excellent tool.

**Comment by pigmonkey** [comment]
Thanks. Fixed, I think.
